container/sagemaker/serve.py

```diff
@@ -26,10 +26,17 @@ def __init__(self):
         self._state = 'initializing'
         self._nginx = None
         self._tfs = None
+        self._requirements = None
+        self._execute_custom_code = False
+        self._python_service = None
+        self._nginx_ping_requests = 'js_content ping'
+        self._nginx_invocation_requests = 'js_content invocations'
         self._tfs_version = os.environ.get('SAGEMAKER_TFS_VERSION', '1.12')
         self._nginx_http_port = os.environ.get('SAGEMAKER_BIND_TO_PORT', '8080')
         self._nginx_loglevel = os.environ.get('SAGEMAKER_TFS_NGINX_LOGLEVEL', 'error')
         self._tfs_default_model_name = os.environ.get('SAGEMAKER_TFS_DEFAULT_MODEL_NAME', None)
+        self._python_path = os.environ.get('PYTHONPATH', '/opt/ml/model/lib')
+        self._gunicorn_port = os.environ.get('GUNICORN_PORT', 5000)
```

Comment:
users shouldn't need to set this... the container should just pick a port within the safe port range and use it.